src/content/pages/forms.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Forms"
+description: "this is meta description"
+draft: false
+---
+
+#### [Reimbursement Form](https://form.jotform.com/Troop150/troop-150-reimbursement-form)
```

Comment:
I think we should put this at the bottom of the list. I think the health forms are more commonly used by troop members.